Makefile

```diff
@@ -81,12 +98,33 @@ distclean:
 	-rm -rf deps/icu
 	-rm -rf deps/icu4c*.tgz deps/icu4c*.zip deps/icu-tmp
 	-rm -f $(BINARYTAR).* $(TARBALL).*
+	-rm -rf deps/v8/testing/gmock
+	-rm -rf deps/v8/testing/gtest
 
 check: test
 
 cctest: all
 	@out/$(BUILDTYPE)/$@
 
+v8:
+	tools/make-v8.sh v8
+	#cd deps/v8
+ifneq (,$(filter $(DESTCPU),x86))
+	+make -C deps/v8 $(V8_BUILD_NO_I18N);
+else
+ifneq (,$(filter $(ARCH),x86))
+	+make -C deps/v8 $(V8_BUILD_NO_I18N);
+else
+ifeq ($(ARCH)$(DESTCPU),)
+	+make -C deps/v8 $(V8_BUILD_NO_I18N);
+else
+	+make -C deps/v8 $(ARCH) $(V8_BUILD_NO_I18N);
+endif
+	+make -C deps/v8 $(ARCH) $(V8_BUILD_NO_I18N);
+endif
+	+make -C deps/v8 $(ARCH) $(V8_BUILD_NO_I18N);
```

Comment:
Yeah, you need `$(MAKE)` here to avoid all kinds of redirects where gmake != make.